experiment/bootstrap/main.go

```diff
@@ -30,8 +30,8 @@ const (
 	CloudSDKEnv            string = "CLOUDSDK_CONFIG"
 	GCEPrivKeyEnv          string = "JENKINS_GCE_SSH_PRIVATE_KEY_FILE"
 	GCEPubKeyEnv           string = "JENKINS_GCE_SSH_PUBLIC_KEY_FILE"
-	AWSPrivKeyEnv          string = "JENKINS_AWS_SSH_PRIVATE_KEY_FILE"
-	AWSPubKeyEnv           string = "JENKINS_AWS_SSH_PUBLIC_KEY_FILE"
+	AWSPrivKeyEnv          string = "AWS_SSH_PRIVATE_KEY_FILE"
```

Comment:
heh reminder to myself delete this experiment. we're definitely not going this route